bzt/modules/aggregator.py

```diff
@@ -71,6 +74,7 @@ def __init__(self, perc_levels=(), rt_dist_maxlen=None):
         self.get(self.ERRORS, [])
         self.get(self.RESP_TIMES, Counter())
```

Comment:
We'll have to cleanup old way, because there is a lot of code spends time on it.